legal-api/src/legal_api/reports/report.py

```diff
@@ -169,6 +169,14 @@ def _get_template_data(self, report_type=None):  # pylint: disable=too-many-bran
                 self._set_directors(filing)
             except KeyError:
                 pass
+
+            # set translation list for COA/COD/Alteration filing
+            try:
+                # FUTURE: get current name translations from database
+                filing['listOfTranslations'] = filing['business'].get('nameTranslations', [])
```

Comment:
Could you please remove this block? Only the changes for IA needs to be present in this PR

Reply:
Is this not needed for (near-future) Notice Of Articles outputs for (BCOMP) COA, COD and Alteration filings?

This other ticket specifies a NOA for these other filings: /bcgov/entity#4107

Reply:
I will remove this code as requested, as it looks this code will be reworked as part of 4299 (ie, to fetch additional output data that is not in the filing JSON).

Reply:
Notice of Article is going to be a totally different approach. For incorporation flow, the IA and NOA make use of the same template with some if conditions